readme.md

```diff
@@ -18,7 +18,7 @@
 
 ## Description
 
-By utilizing a simple and minimal usage syntax, that requires a flat learning curve, taskbook enables you to effectively manage your tasks and notes across multiple boards from within your terminal. All data are written atomically to the storage in order to prevent corruptions, and are never shared with any third party entities. Deleted items are automatically archived and can be inspected or restored at any moment.
+By utilizing a simple and minimal usage syntax, that requires a flat learning curve, taskbook enables you to effectively manage your tasks and notes across multiple boards from within your terminal. All data is written automatically to the storage in order to prevent corruptions, and are never shared with any third party entities . Deleted items are automatically archived and can be inspected or restored at any moment.
```

Comment:
This is too much of a trivial change, both can be considered correct.
Thank you for opening the pull request and initializing the discussion : )